clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/evaluation/EventAdapter.kt

```diff
@@ -81,11 +81,20 @@ class EventAdapter(
         return items
             .filterNotNull()
             .map { productMap: Map<String, Any> ->
-                val normalisedMap = productMap.map {
-                    Utils.getNormalizedName(it.key) to it.value
-                }.toMap()
 
-                TriggerValue(normalisedMap[Utils.getNormalizedName(propertyName)])
+                var op = productMap[propertyName]
```

Comment:
What is the fullform of op?

Reply:
it is just a local var to capture trigger value returned for the key.